src/Shared/CoreCLRAssemblyLoader.cs

```diff
@@ -63,11 +72,29 @@ public Assembly LoadFromPath(string fullPath)
             }
         }
 
+        private Assembly TryGetWellKnownAssembly(AssemblyLoadContext context, AssemblyName assemblyName)
```

Comment:
Hah, but I didn't complain about `TryResolveAssembly` below. Consistency is good.